README.md

```diff
@@ -8,6 +8,62 @@ a threshold.
 The ultimately use case for this script is to run it periodically and use it to
 trigger alerts/notifications that the Product Delivery team can take heed of.
 
+## Architectural Overview
+
+### In Plain English
+
+This is stateless script, written in Python, meant to be executed roughly once a day.
+It uses `requests.get()` to query [Redash](https://sql.telemetry.mozilla.org) for
+a query's data that is pre-emptively run every 24 hours. The data it analyses is a
+list of every collection stored in Remote Settings (Kinto) along with a count of
```

Comment:
Actually, it targets all the components using the unified Telemetry https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/uptake.html   Remote Settings is one of them. 

Since the default config filters out the other components, I don't know if we should mention that

Reply:
I don't your point. `histogram_parent_uptake_remote_content_result_1` is hardcoded in the Redash query. So how is my sentence incorrect?

Reply:
The unified Uptake Telemetry is not specific to Remote Settings. Proof is that Shield and Normandy use it.